src/assets/scripts/client/App.js

```diff
@@ -62,6 +63,9 @@ export default class App {
      */
     createHandlers() {
         this.eventBus.on(EVENT.PAUSE_UPDATE_LOOP, this.updateRun);
```

Comment:
should this be in an `.enable()` method?

If it stays here, will need a blank line below to separate from the property assignments

Reply:
it should, yes, but I have a hunch we use it before that method is called and I wanted to keep the changes here directed at our issue.